build/accelerate_launch.py

```diff
@@ -43,10 +47,17 @@
     USER_ERROR_EXIT_CODE,
     INTERNAL_ERROR_EXIT_CODE,
 )
+from tuning.data import tokenizer_data_utils
 
 ERROR_LOG = "/dev/termination-log"
 
 
+def get_base_model_from_adapter_config(adapter_config):
+    with open(adapter_config, "r", encoding="utf-8") as config_file:
```

Comment:
Can you add a docstring to this function, just explaining that the adapter config is for peft models since this might be confusing to people that have only used transformers models?